website/content/en/docs/building-operators/ansible/development-tips.md

```diff
@@ -192,6 +192,10 @@ if there are many watched resources. Typically, this option should only be used
 
 ### Testing an Ansible operator locally
 
+**Prerequisites**: Ensure that [Ansible Runner][ansible-runner-tool] and [Ansible Runner
+HTTP Plugin][ansible-runner-http-plugin] is installed or else you will see
```

Comment:
Broken links? I think we need to add the reference links for `ansible-runner-tool` and `ansible-runner-http-plugin`.

![image](https://user-images.githubusercontent.com/6660644/88851223-525d2f80-d1a1-11ea-8805-ac8d88382469.png)

Reply:
Is the `Prerequisites*` specifics for this subsection. If yes. Great .. it will make it easier when we try to split out all if not would be nice have on top of the doc.

Reply:
Yes these tools are only required to run the operator locally. In a container they are provided by the base image.